packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBar.ios.js

```diff
@@ -57,13 +65,13 @@ const IOS_THEME_ATTRIBUTES = [
 const statusBarPadding = isIphoneX ? 44 : 20;
 
 // NOTE: this file explicitly does not use the Backpack tokens(for spacing) because it's based on UIKit design tokens not Backpack.
-const styles = StyleSheet.create({
+const dynamicStyles = BpkDynamicStyleSheet.create({
   barOuter: {
     flexDirection: 'column',
     paddingHorizontal: 8, // eslint-disable-line backpack/use-tokens
     width: '100%',
-    backgroundColor: colorSkyGrayTint07,
-    shadowColor: colorSkyGrayTint06,
+    backgroundColor: backgroundSecondaryColor,
+    shadowColor: { light: colorSkyGrayTint06, dark: colorBlackTint02 },
```

Comment:
I don't think we should have shadow in dark mode. This PR will allow us to provide `null` here https://github.com/Skyscanner/backpack-react-native/pull/296. 

Not a blocker I believe.